src/Hamiltonians/excitations.jl

```diff
@@ -195,7 +195,7 @@ function transcorrelated_three_body_excitation(add_a, add_b, i, map_a, map_b)
         return add_a, add_b, 0.0, k,l
     end
     p_k_index, q_l_index = find_mode(add_a, (p_k, q_l))
-    s_kl_index = find_mode(add_a, s_kl)
+    s_kl_index = find_mode(add_b, s_kl)
```

Comment:
Was that a bug in transcorrelated? Does it have consequences?

Reply:
It's a bug, but thankfully it doesn't have consequences. If an address is represented by a `BitString`, the occupation number is taken directly from the address - the computed index was wrong, but the information contained in it wasn't actually used.